common/memops.c

```diff
@@ -146,6 +153,16 @@
 		(d) = f_round ((s) * SAMPLE_24BIT_SCALING);                    \
 	}
 
+#define float_32(s, d)                                                 \
```

Comment:
This whole if-else block needs to be wrapped inside `do { ... } while(0)` to ensure that using it behaves like a function call would.

Reply:
agreed, I kept it like this for consitency with the other macros.
Personally I'd forgo the macro all together